libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/RtspMediaTrack.java

```diff
@@ -160,6 +169,24 @@ private static void processAacFmtpAttribute(
             AacUtil.buildAacLcAudioSpecificConfig(sampleRate, channelCount)));
   }
 
+  private static void processMPEG4FmtpAttribute(
+      Format.Builder formatBuilder, ImmutableMap<String, String> fmtpAttributes) {
+    @Nullable String configInput = fmtpAttributes.get(PARAMETER_CONFIG);
+    if (configInput != null) {
+      byte[] csd = Util.getBytesFromHexString(configInput);
+      ImmutableList<byte[]> initializationData = ImmutableList.of(csd);
+      formatBuilder.setInitializationData(initializationData);
+      Pair<Integer, Integer> dimensions = CodecSpecificDataUtil.parseMpeg4VideoSpecificConfig(csd);
+      formatBuilder.setWidth(dimensions.first);
+      formatBuilder.setHeight(dimensions.second);
+    }
+    @Nullable String profileLevel = fmtpAttributes.get(PARAMETER_PROFILE_LEVEL_ID);
+    if (profileLevel == null) {
+      profileLevel = "1"; // default
+    }
+    formatBuilder.setCodecs(MPEG4_CODECS_PREFIX + profileLevel);
```

Comment:
Maybe do a ternary here and remove the if: 

`formatBuilder.setCodecs(MPEG4_CODECS_PREFIX + profileLevel == null ? "1" : profileLevel);`